plugin/db_ui.vim

```diff
@@ -115,6 +115,7 @@ augroup END
 
 command! DBUI call db_ui#open('<mods>')
 command! DBUIToggle call db_ui#toggle()
+command! DBUIQuit call db_ui#quit()
```

Comment:
Lets call it `DBUIClose` and the function `db_ui#close()`, just to have it properly as an opposite of `open`.